autoload/http.vim

```diff
@@ -151,12 +151,26 @@ endfunction
 function! s:new_response_buffer(request_buffer, response) abort
     let l:request_buffer_name  = bufname(a:request_buffer)
     let l:buffer_name = fnamemodify(l:request_buffer_name, ":r") . '.response.' . localtime() . '.http'
-    if g:vim_http_split_vertically
-      execute 'vert new ' . l:buffer_name
-    else
-      execute 'new ' . l:buffer_name
-    end
+    if g:vim_http_tempbuffer
+      for win in range(1, winnr('$'))
+        if getwinvar(win, 'temp')
+          execute win . 'windo close'
+        endif
+      endfor
+    endif
+    " if g:vim_http_split_vertically
+    "   execute 'vert new ' . l:buffer_name
+    " else
+    "   execute 'new ' . l:buffer_name
+    " end
```

Comment:
You can go ahead and remove this stuff, we have version control if we ever need it back.